database/mongo/src/main/java/org/geysermc/floodgate/database/MongoDbDatabase.java

```diff
@@ -335,13 +342,4 @@ private UUID bytesToUUID(String uuidBytes) {
         return new UUID(buf.getLong(), buf.getLong());
     }
 
-    public boolean collectionNotExists(final String collectionName) {
-        try (MongoCursor<String> collectionNames = database.listCollectionNames().cursor()) {
-            if (collectionNames.hasNext() && collectionNames.next().equals(collectionName)) {
-                return false;
-            }
-        }
-        return true;
-    }
```

Comment:
In my opinion, this PR would be better if you were to just fix this function instead of all the try, catch blocks.

Reply:
Ok I fixed the function directly, this is better now